posts/2019/05/quansight-partners-with-quansight-to-sponsor-spyder.md

```diff
@@ -0,0 +1,112 @@
+<!--
+.. title: Quansight Partners with Quansight to sponsor Spyder
+.. slug: quansight-partners-with-quansight-to-sponsor-spyder
+.. date: 2019-05-16 21:22:58 UTC-05:00
+.. tags: Spyder
+.. category: 
+.. link: 
+.. author: Gonzalo Peña-Castellanos
+.. description: 
+.. type: text
+-->
+
+TDK-Micronas is sponsoring Spyder development efforts through Quansight Labs.
+This will enable the development of some features that have been requested by
+our users, as well as new features that will help TDK develop custom Spyder
+plugins in order to complement their Automatic Test Equipment (ATE’s) in the
+development of their Application Specific Integrated Circuits (ASIC’s).
+
+## Multiple Projects aka Workspaces
+
+In the current state Spyder can only handle one active project at a time.
+Although in the past we had basic support for workspaces, it was never a fully
+functional feature, so to ease development and simplify the user experience,
+we decided to remove it on the 3.x series.
+
+For TDK-Micronas and within its ATE tooling development as well as for a large
```

Comment:
I'd remove "and within its ATE tooling development", it doesn't quite fit and isn't really necessary here